src/components/project/donationsTable.js

```diff
@@ -269,11 +287,18 @@ const CustomTable = () => {
                     data-label='Donor'
                     sx={{ variant: 'text.small', color: 'secondary' }}
                   >
-                    <Avatar src='https://www.filepicker.io/api/file/4AYOKBTnS8yxt5OUPS5M' />
+                    <Avatar
+                      src={
+                        i?.extra?.userByAddress?.avatar ||
+                        'https://www.filepicker.io/api/file/4AYOKBTnS8yxt5OUPS5M'
```

Comment:
Earmarking this as being hardcoded

Reply:
@markoprljic do you recommend a default avatar for users that doesn't have profile pic that isn't a cute xmas puppy or should I go with the boring facebook guy in shades? 😃